src/System.Net.WebHeaderCollection/src/System/Net/HeaderInfoTable.cs

```diff
@@ -10,64 +10,137 @@ namespace System.Net
+                new HeaderInfo(HttpKnownHeaderNames.IfModifiedSince,    true,   false,  false,  SingleParser),
+                new HeaderInfo(HttpKnownHeaderNames.IfUnmodifiedSince,  false,  false,  false,  SingleParser),
+                new HeaderInfo(HttpKnownHeaderNames.KeepAlive,          false,  true,   false,  SingleParser),
+                new HeaderInfo(HttpKnownHeaderNames.Location,           false,  false,  false,  SingleParser),
+                new HeaderInfo(HttpKnownHeaderNames.LastModified,       false,  false,  false,  SingleParser),
+                new HeaderInfo(HttpKnownHeaderNames.MaxForwards,        false,  false,  false,  SingleParser),
+                new HeaderInfo(HttpKnownHeaderNames.Pragma,             false,  false,  true,   MultiParser),
+                new HeaderInfo(HttpKnownHeaderNames.ProxyAuthenticate,  false,  false,  true,   MultiParser),
+                new HeaderInfo(HttpKnownHeaderNames.ProxyAuthorization, false,  false,  true,   MultiParser),
+                new HeaderInfo(HttpKnownHeaderNames.ProxyConnection,    true,   false,  true,   MultiParser),
+                new HeaderInfo(HttpKnownHeaderNames.Range,              true,   false,  true,   MultiParser),
+                new HeaderInfo(HttpKnownHeaderNames.Referer,            true,   false,  false,  SingleParser),
+                new HeaderInfo(HttpKnownHeaderNames.RetryAfter,         false,  false,  false,  SingleParser),
+                new HeaderInfo(HttpKnownHeaderNames.Server,             false,  false,  false,  SingleParser),
+                new HeaderInfo(HttpKnownHeaderNames.SetCookie,          false,  false,  true,   SetCookieParser),
+                new HeaderInfo(HttpKnownHeaderNames.SetCookie2,         false,  false,  true,   SetCookieParser),
+                new HeaderInfo(HttpKnownHeaderNames.TE,                 false,  false,  true,   MultiParser),
+                new HeaderInfo(HttpKnownHeaderNames.Trailer,            false,  false,  true,   MultiParser),
+                new HeaderInfo(HttpKnownHeaderNames.TransferEncoding,   true,   true,   true,   MultiParser),
+                new HeaderInfo(HttpKnownHeaderNames.Upgrade,            false,  false,  true,   MultiParser),
+                new HeaderInfo(HttpKnownHeaderNames.UserAgent,          true,   false,  false,  SingleParser),
+                new HeaderInfo(HttpKnownHeaderNames.Via,                false,  false,  true,   MultiParser),
+                new HeaderInfo(HttpKnownHeaderNames.Vary,               false,  false,  true,   MultiParser),
+                new HeaderInfo(HttpKnownHeaderNames.Warning,            false,  false,  true,   MultiParser),
+                new HeaderInfo(HttpKnownHeaderNames.WWWAuthenticate,    false,  true,   true,   SingleParser)
             };
 
             HeaderHashTable = new Hashtable(InfoArray.Length * 2, CaseInsensitiveAscii.StaticInstance);
```

Comment:
Aside: At one point before the dead code was deleted, this had been updated to use `Dictionary<string, HeaderInfo>` and was initialized more efficiently. But it was added back as `Hashtable`. In a separate PR it'd be nice to move this back to Dictionary. See https://github.com/dotnet/corefx/blob/ea8f322ffe7fe738e77a6a5ea831596eba32287e/src/System.Net.WebHeaderCollection/src/System/Net/HeaderInfoTable.cs